capitals.rb

```diff
@@ -151,3 +151,75 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+@states.each do | state |
+    state[:correct] = 0
+    state[:wrong] = 0
+end
+
+def shuffle_states
+    @states = @states.shuffle
+end
+
+def reset
+    @states_remaining = 50
+    @correct_answers = 0
+    @incorrect_answers = 0
+    puts "Welcome to \"Learn Our State Capitals\"!\n The objective of the game is to guess the correct state capital for the corresponding state.\n The game will keep a running tally of your correct and incorrect answers.\n Ready?\n Let's Plays!\n\n"
+
+end
+
+def final_tally
+    puts "Correct Answers: #{@correct_answers}"
+    puts "Incorrect Answers: #{@incorrect_answers}\n\n"
+end
+
```

Comment:
this `update` method is my favorite part. feel like you might've been digging deep with functional programming stuff. Was this influenced by [ELM](courses.knowthen.com/p/elm-for-beginners)?